app/services/service_offering/icons.rb

```diff
@@ -1,21 +0,0 @@
-module ServiceOffering
-  class Icons
-    attr_reader :icon_id
-    attr_reader :icon
-
-    def initialize(id)
-      @icon_id = id.to_s
-    end
-
-    def process
-      TopologicalInventory.call do |api_instance|
```

Comment:
@lindgrenj6 there is no need to look up the icon from Topology? I understand why this service class is gone but is this lookup no longer needed as well?

Reply:
Yeah, I should have removed that class a long time ago when we first started importing the icons into catalog. 

We do the call to topo here now: https://github.com/lindgrenj6/catalog-api/blob/ecda01f6dc1ba0337704da1d5d5d97debd709ef5/app/services/service_offering/add_to_portfolio_item.rb#L54